pkg/collector/reconcile/horizontalpodautoscaler.go

```diff
@@ -119,13 +120,23 @@ func setAutoscalerSpec(params Params, autoscalingVersion autodetect.AutoscalingV
 			} else {
 				updated.(*autoscalingv2beta2.HorizontalPodAutoscaler).Spec.MinReplicas = &one
 			}
+			if params.Instance.Spec.TargetCPUUtilization != nil {
```

Comment:
Is this check needed? The defaulting webhook should make sure the value is always set.

Reply:
Removed.